### PR TITLE
Remove react hook form in send email action

### PR DESCRIPTION
**Pull request by thomtrp**
- remove react hook form
- put back multiline for body

### CODE REVIEW

File: packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormSendEmail.tsx [hunk 1]
```diff
@@ -79,14 +76,13 @@ export const WorkflowEditActionFormSendEmail = ({
...
-    form.setValue('subject', action.settings.input.subject ?? '');
-    form.setValue('body', action.settings.input.body ?? '');
-  }, [action.settings, form]);
+    setFormData({
+      connectedAccountId: action.settings.input.connectedAccountId,
+      email: action.settings.input.email,
+      subject: action.settings.input.subject ?? '',
+      body: action.settings.input.body ?? '',
+    });
+  }, [action.settings.input]);
```
Comment (Devessier) [inline packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormSendEmail.tsx:85]: Honestly, I'm not sure if we need this useEffect. If the server takes time to respond, the form's state might be corrupted.

Comment (Devessier) [inline packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormSendEmail.tsx:85]: After reviewing this, I would remove the useEffect. We built the form field inputs with a `defaultValue` in mind, not a `controlled` `value`. Technically, our abstractions don't allow us to set the value of TipTap editors.

This `useEffect` has no effect, at least for form fields. The connected account select is a controlled component, and this `useEffect` would set its value. I think it's better to consider that the back-end state doesn't override the local state. We'll see how we can handle it properly if that's okay with you. Preventing the local override makes the experience on a slow network better. I prefer not to override what people might have typed in inputs to display the fresher data coming from the backend.

**Review by Devessier — CHANGES_REQUESTED** [review]
It looks good! Thank you for taking this bug. I would like us to delete the useEffect we used to synchronize the local state with the backend state. It's primarily dead code, and it's simpler not to override the local state with the backend state for now.

File: packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormSendEmail.tsx (hunk 1 above)
Comment (martmull) [inline packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormSendEmail.tsx:85]: +1 for removing the useEffect. I agree with Baptiste on prioritizing optimistic rendering over matching the backend state. More generally, removing a useEffect aligns with the project standards.

**Review by martmull — APPROVED** [review]
Agree with Baptiste comment, I still approve

**Review by Devessier — APPROVED** [review]
Nice! Thank you.

**Pull request merged**